manifest_template.yml

```diff
@@ -12,13 +12,17 @@ APIGEE_ENVIRONMENTS:
     display_name: Internal QA Sandbox
   - name: ref
     display_name: Reference
+    ratelimit: '210000pm'
```

Comment:
Why is this in quotes and nothing else is? Does it get parsed differently?

Why does it need two of the same value in different fields?

Why do we only rate limit these environments and not every environment?